app/views/api/v2/hosts/main.json.rabl

```diff
@@ -37,6 +37,10 @@ node :hostgroup_name do |host|
   host.hostgroup.name if host.hostgroup.present?
 end
 
+node :operatingsystem_icon do |host|
+  icon(@host.operatingsystem, path: true) if host.operatingsystem.present?
+end
+
```

Comment:
I don't think this should be in the API - it is only used by the UI, and doesn't make sense for users of the API. 
Since the icon is calculated anyways, perhaps we should create a front-end version of https://github.com/theforeman/foreman/blob/develop/app/helpers/operatingsystems_helper.rb#L9-L71 ?

Reply:
I created a front-end version of it :)